lib/sisimai/data.rb

```diff
@@ -110,7 +110,7 @@ def self.make(data: nil, **argvs)
       rfc822data = messageobj.rfc822
       fieldorder = { :recipient => [], :addresser => [] }
       objectlist = []
-      rxcommands = %r/\A(?:EHLO|HELO|MAIL|RCPT|DATA|QUIT)\z/
+      commandset = ['EHLO', 'HELO', 'MAIL', 'RCPT', 'DATA', 'QUIT']
```

Comment:
__[RuboCop]__

Style/WordArray: Use `%w` or `%W` for an array of words.


<!-- Comment by [SideCI](https://sideci.com) -->